### PR TITLE
Adding Diff IQ Measurement Class

### LINKED ISSUE #64: Add Differential IQ Measurement Class
Marki requested that Diff IQ to be implemented as they use it regularly in production

### PR DESCRIPTION
**Pull request by c-montes**
Added DiffIQ measurement class and provided support to customize Frequency Ranges, Receivers, etc by using AvailableValues

Close #64

### CODE REVIEW

File: OpenTap.Plugins.PNAX/General/Differential IQ/DIQChannel.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
+﻿// Author: CMontes
+// Copyright:   Copyright 2023 Keysight Technologies
```
Comment (alan-copeland-keysight) [inline OpenTap.Plugins.PNAX/General/Differential IQ/DIQChannel.cs:2]: All the copyrights should probably be updated with 2024:
`// Copyright:   Copyright 2023-2024 Keysight Technologies`

File: OpenTap.Plugins.PNAX/Instrument/PNAModulationDistortion.cs [hunk 2]
```diff
@@ -197,7 +197,19 @@ public enum MODTableSetupEVMEnum
         EVMDistUn21 = 8,
         [Display("EVM Norm")]
         [Scpi("EVM Norm")]
-        EVMNorm = 16
+        EVMNorm = 16,
```
Comment (alan-copeland-keysight) [inline OpenTap.Plugins.PNAX/Instrument/PNAModulationDistortion.cs:200]: Does this change to an enum constitute a "breaking change"? At the very least this should be a "minor version" bump, not just a patch release. What are your thoughts @c-montes / @alexwang-keysight / @bdirenzo-keysight / @alnlarsen / @rmadsen-ks ?

**Review by alan-copeland-keysight — COMMENTED** [review]
Copyrights should be updated. The other question is definitely a question that I don't know the right answer to.

**Review by alexwang-keysight — APPROVED** [review]

File: OpenTap.Plugins.PNAX/General/Differential IQ/DIQChannel.cs (hunk 1 above)
Comment (c-montes) [reply OpenTap.Plugins.PNAX/General/Differential IQ/DIQChannel.cs]: Thanks, I updated all files with 2024

File: OpenTap.Plugins.PNAX/Instrument/PNAModulationDistortion.cs (hunk 2 above)
Comment (c-montes) [reply OpenTap.Plugins.PNAX/Instrument/PNAModulationDistortion.cs:200]: I added additional options for this enum based on a request from the IST, this was part of the Modulation Distortion implementation I completed last week. Nobody is using this besides the IST for now.

**Pull request merged**